common/types.go

```diff
@@ -168,3 +169,71 @@ type LogConfig struct {
 	FileCount   uint64
 	TimeTracker bool
 }
+
+type Block struct {
+	Id         string
```

Comment:
As there are many blocks stored in the system lets arrange the fields in bye order, put all 64 bit fields before the name to conserve memory.

Reply:
same applies for all structures which will be stored in bulk